.github/workflows/sync-from-public.yml

```diff
@@ -0,0 +1,50 @@
+name: Daily Sync from Public Repo
+
+on:
+  schedule:
+    - cron: '0 8 * * *'  # Daily at 8:00 AM UTC
+  workflow_dispatch:
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Configure Git
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+
+      - name: Sync main with public repository
+        run: |
+          git remote add public https://github.com/anthropics/claude-cookbooks.git
+          git fetch public
+          git checkout main
+
+          # Safety check: verify we're on main branch
+          CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+          if [ "$CURRENT_BRANCH" != "main" ]; then
+            echo "❌ ERROR: Not on main branch (currently on: $CURRENT_BRANCH)"
+            exit 1
+          fi
+
+          # Try fast-forward first (safest and expected case)
+          if git merge public/main --ff-only; then
+            echo "✓ Fast-forwarded to public/main"
+            git push origin main:main
+          else
+            echo "⚠️  Cannot fast-forward. Private main has diverged from public."
+            echo "ℹ️  Resetting private main to match public main exactly..."
```

Comment:
hmm are we sure we want to do this here? it seems like this would be destructive to any changes made by Ants?